test/test_helper.rb

```diff
@@ -27,7 +27,11 @@
 require './api/root'
 
 require 'minitest/reporters'
-Minitest::Reporters.use!
+Minitest::Reporters.use! [
+  Minitest::Reporters::ProgressReporter.new,
+  ENV['HTML'] && Minitest::Reporters::HtmlReporter.new, # Create html report with many more informations
+  ENV['TIME'] && Minitest::Reporters::SpecReporter.new, # Generate a report to find slowest tests
```

Comment:
```suggestion
  ENV['HTML']==true && Minitest::Reporters::HtmlReporter.new, # Create html report with many more informations
  ENV['TIME']==true && Minitest::Reporters::SpecReporter.new, # Generate a report to find slowest tests
```
don't know if ENV[''] is a string, in that case the above code should be `ENV['..']=='true' `

Reply:
none of these work, I’ll have to take a closer look